### ISSUE TITLE
Inconsistency between pip version (3.8.2) and installed version (3.8.1)

### ISSUE THREAD

**Issue by dpasqualin** [RaRe-Technologies/gensim#2796]
#### Problem description
The gensim package version is 3.8.2 on [pip](https://pypi.org/project/gensim/), but after installing it and checking the version on python console I see 3.8.1:
```python
import gensim
gensim.__version__
# '3.8.1'
```

#### Versions
```bash
Linux-5.3.0-46-generic-x86_64-with-glibc2.10
Python 3.8.2 (default, Mar 26 2020, 15:53:00) 
[GCC 7.3.0]
NumPy 1.18.1
SciPy 1.4.1
gensim 3.8.1
FAST_VERSION 1
```

**Comment by piskvorky** [comment]
I can replicate this – definitely a bug. Thanks for reporting. CC @mpenkov

**Comment by mpenkov** [comment]
Fixed in a8e771da3cdf1856307edc647cc452f7d3c34d19

**Comment by XiaoyanZhan201** [comment]
I am facing the same problem when installing gensim 3.8.2 using pip.
gensim-3.8.2-cp36-cp36m-manylinux1_x86_64.whl

Versions:
Python 3.6.10
numpy 1.14.3

**Comment by piskvorky** [comment]
Yes, the problem was fixed in 3.8.3.
